beacon_node/beacon_chain/src/test_utils.rs

```diff
@@ -2371,7 +2371,11 @@ where
 
         // Blobs are stored as data columns from Fulu (PeerDAS)
         if self.spec.is_peer_das_enabled_for_epoch(block.epoch()) {
-            let columns = self.chain.get_data_columns(&block_root).unwrap().unwrap();
+            let columns = self
+                .chain
+                .get_data_columns(&block_root)
+                .unwrap()
+                .or_default();
```

Comment:
I think we cant default this - in this case the node should have columns in the db because we checked the block has blobs.